config/environments/development.rb

```diff
@@ -1,4 +1,13 @@
 Rails.application.configure do
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_credentials: {
+    bucket: ENV["AWS_BUCKET"],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.